src/commcare_cloud/commands/terraform/migrations/0002_name_servers.py

```diff
@@ -0,0 +1,17 @@
+def get_new_resource_address(environment, old_resource_address):
+    import re
+    server_matcher = re.compile(r'^module.servers.aws_instance.server\[(\d+)\]$')
+
+    match = server_matcher.match(old_resource_address)
+    if match:
+        index = int(match.group(1) or 0)
+        server_config = environment.terraform_config.servers[index]
+        return 'module.server__{}.aws_instance.server'.format(server_config.server_name)
+
+    if old_resource_address == 'module.proxy_servers.aws_instance.server':
+        server_config = environment.terraform_config.proxy_servers[0]
+        return 'module.proxy_server__{}.aws_instance.server'.format(server_config.server_name)
+
+    if old_resource_address == 'aws_eip.proxy':
+        server_config = environment.terraform_config.proxy_servers[0]
+        return 'aws_eip.{}-production'.format(server_config.server_name)
```

Comment:
Is this function supposed to return `None` if none of the conditions match? We often do not explicitly `return None` (since Python does that implicitly), but I think it's a good practice to make the intention clear.

As a rule, a function should always explicitly return some value if any return statements with a value exist in the function. This makes it easier for a code reviewer to detect if a mistake has been made.

Reply:
Good suggestion. Decided to roll it up into this same PR in the following two new commits. Let me know what you think